requirements.txt

```diff
@@ -1,2 +1,2 @@
-csp-billing-adapter
+csp-billing-adapter @ git+https://github.com/suse-enceladus/csp-billing-adapter@devel
```

Comment:
since we are close to merging devel to main should we reflect that here ? @rtamalin thoughts?

Reply:
Eventually we could push these to PyPI as they are open source site packages. Then reference the name only. But that's low priority.

Reply:
That should work though I have run into issues in the past getting the syntax correct for such requirements such that they work correctly for pip install -r req.txt and for use when the contents of those files are pulled into the setup.py as the package requirements specification... It may be simpler/safer to just get the devel branch merged to main in SUSE-Enceladus/csp_billing_adapter.

Reply:
Updated to main